### PR TITLE
TIP visual content

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Update TIP related content for the release:
- Point to video url
- Take Quick Start animations from TIP docs

### CODE REVIEW

File: services/web/client/source/class/osparc/component/tutorial/ti/ElectrodeSelector.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ qx.Class.define("osparc.component.tutorial.ti.ElectrodeSelector", {
       const label2 = osparc.component.tutorial.ti.Slides.createLabel(text2);
       this._add(label2);
 
-      const image = new qx.ui.basic.Image("osparc/tutorial/ti/ElectrodeSelector.gif").set({
+      const image = new qx.ui.basic.Image("https://itisfoundation.github.io/ti-planning-tool-manual/assets/quickguide/electrode_selector.gif").set({
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/component/tutorial/ti/ElectrodeSelector.js:41]: IMO this is indeed a far better and more sustainable approach! Great!

**Review by pcrespov — APPROVED** [review]

File: services/web/client/source/class/osparc/component/tutorial/ti/ElectrodeSelector.js (hunk 1 above)
Comment (odeimaiz) [reply services/web/client/source/class/osparc/component/tutorial/ti/ElectrodeSelector.js:41]: 👍

**Review by sanderegg — APPROVED** [review]
should these things not go in something configurable instead of hard-coded?

**Comment by odeimaiz** [comment]
> should these things not go in something configurable instead of hard-coded?

Ideally yes.

**Pull request merged**